dice_ml/data_interfaces/private_data_interface.py

```diff
@@ -182,6 +192,12 @@ def get_indexes_of_features_to_vary(self, features_to_vary='all'):
                     ixs.append(colidx)
             return ixs
 
+    def from_label(self, data):
```

Comment:
add a string documentation for this function just like from_dummies function below.

That holds for every new function that you add.